src/js/experimental/media-chrome-listbox.js

```diff
@@ -5,42 +5,39 @@ const template = document.createElement('template');
 
 template.innerHTML = /*html*/`
 <style>
-  :host ul {
+  :host {
     font: var(--media-font,
       var(--media-font-weight, normal)
       var(--media-font-size, 1em) /
       var(--media-text-content-height, var(--media-control-height, 24px))
       var(--media-font-family, helvetica neue, segoe ui, roboto, arial, sans-serif));
     color: var(--media-text-color, var(--media-primary-color, rgb(238 238 238)));
     background: var(--media-listbox-background, var(--media-control-background, var(--media-secondary-color, rgb(20 20 30 / .8))));
-    list-style: none;
     display: inline-flex;
     flex-direction: column;
     gap: 0.5em;
     margin: 0;
     padding: 0.5em;
   }
 
-  ::slotted(media-chrome-listitem[tabindex="0"]:focus-visible),
-  media-chrome-listitem[tabindex="0"]:focus-visible {
+  ::slotted(media-chrome-option[tabindex="0"]:focus-visible),
+  media-chrome-option[tabindex="0"]:focus-visible {
     box-shadow: inset 0 0 0 2px rgb(27 127 204 / .9);
     outline: 0;
   }
 
-  ::slotted(media-chrome-listitem[aria-selected="true"]),
-  media-chrome-listitem[aria-selected="true"] {
+  ::slotted(media-chrome-option[aria-selected="true"]),
+  media-chrome-option[aria-selected="true"] {
     background-color: var(--media-listbox-selected-background, rgb(122 122 184 / .8));
   }
 
-  ::slotted(media-chrome-listitem:hover),
-  media-chrome-listitem:hover {
+  ::slotted(media-chrome-option:hover),
+  media-chrome-option:hover {
```

Comment:
Right now hovering on a selected item triggers the hover state, but it shouldn't. We should ignore this when selected.